modules/core/shared/src/main/scala/lucuma/core/model/Target.scala

```diff
@@ -62,12 +64,13 @@ object Target extends WithId('t') with TargetOptics {
     name:          NonEmptyString,
     ephemerisKey:  EphemerisKey,
     sourceProfile: SourceProfile,
+    catalogInfo:   Option[CatalogInfo],
```

Comment:
We have asteriod vs. comet vs. minor/major body in the nonsidereal side, which appears in the horizons key.  The object type overlaps a bit with that.  Would we fill in an `objectType` from the `EphemerisKey`?

Reply:
In fact the `EphemerisKey` is kind of `CatalogInfo` from horizons.  I wonder if we need `SiderealCatalogInfo` and `NonSiderealCatalogInfo`.